src/mechs/hkdf/hkdf.ts

```diff
@@ -0,0 +1,71 @@
+import * as core from "webcrypto-core";
+import { HmacCryptoKey } from "../hmac/key";
+import { HkdfCryptoKey } from "./key";
+import { BufferSourceConverter, CryptoKey } from "webcrypto-core";
+import crypto from "crypto";
+
+export class HkdfProvider extends core.HkdfProvider {
+
+  private normalizeHash(hash: HashAlgorithmIdentifier): Algorithm {
```

Comment:
this function is odd. `webcrypto-core` normalizes algorithm in SubtleCrypto class
https://github.com/PeculiarVentures/webcrypto-core/blob/master/src/subtle.ts#L99

You can cast type like (it's safely)
```ts
(params.hash as Algorithm).name.replace("-", "");
```
https://github.com/PeculiarVentures/webcrypto/blob/master/src/mechs/ec/crypto.ts#L54

Reply:
I wasn't sure what to do since the type `HashAlgorithmIdentifier` can be both the object and the string. I'll remove this and cast.